### DISCUSSION TITLE
Grundig Chassis NT 43GUT9989

### DISCUSSION THREAD

**Discussion by DIGTV** [PredatH0r/ChanSort#316] [Q&A]
Hi, gibt es die Möglichkeit die User  Datei zu editieren?
dtv_cmdb_2.bin im Ordner NT_TV_ChannelDatabase.
Danke

**Comment by PredatH0r** [comment]
Könnten Sie mir bitte eine komplette Senderliste mit allen Dateien zukommen lassen?
Von den "dtv_cmdb_*" Listen gibt es zig verschiedene Varianten mit unterschiedlichem Aufbau, somit kann ich leider keine allgemeine Aussage dazu machen.
